src/main/groovy/com/twcable/gradle/sling/osgi/SlingOsgiBundle.groovy

```diff
@@ -424,6 +424,11 @@ class SlingOsgiBundle {
         def resolved = statuses[3] as Integer
         def installed = statuses[4] as Integer
 
+        if (resolved == 0 && installed == 0) {
+            log.debug "There are no bundles in the \"resolved\" or \"installed\" state"
```

Comment:
No need. Internally it does that for you as part of the `log.debug(..)` call. It's useful to prevent unnecessary object creation, but we're not doing that here.